src/coreclr/nativeaot/Runtime/eventpipe/ep-rt-aot.h

```diff
@@ -1383,12 +1459,10 @@ ep_rt_utf16_to_utf8_string (
         return NULL;
     
     // shipping criteria: no EVENTPIPE-NATIVEAOT-TODO left in the codebase
-    // TODO: Temp implementation that is the reverse of ep_rt_utf8_to_utf16le_string
+    // Simple implementation to create a utf8 string from a utf16 one
```

Comment:
I think this is implementation is basically assuming only ASCII is used. Maybe assert / comment that?